### PR TITLE
Bug 1260989 - Populate applications installedApps with webapps.json r=benfrancis

### PR DESCRIPTION
**Pull request by albertopq**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=43fbc91a7dd9cdf3cf3488ccc1b3a70819925b77

File: apps/system/js/homescreen_window.js [hunk 1]
```diff
@@ -89,12 +89,12 @@
    */
   HomescreenWindow.prototype.setBrowserConfig =
     function hw_setBrowserConfig(_manifestURL) {
-      // FIXME: don't hardcode that.
-      var manifestURL = 'chrome://gaia/content/homescreen/manifest.webapp';
-      this.url = 'chrome://gaia/content/homescreen/index.html#root';
-      var url = new URL(manifestURL);
-      this.origin = url.origin;
-      this.manifestURL = manifestURL;
+      // FIXME: Why is not manifest.webapp stored?
```
Comment (albertopq) [inline apps/system/js/homescreen_window.js]: Restored to what we have in master (the only difference is adding /manifest.webapp)

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=37ac15f4e2a7dd9f21b1e19ee76ba362cc00d48b

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=60a0a5244f7decc4d969710e20bbdfb4a05c1cae

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=ed8e5bcd764ba49c150b903c1a53826c96b35c4d

File: apps/system/js/browser_config_helper.js [hunk 2]
```diff
@@ -2,7 +2,7 @@
 'use strict';
 
 (function(exports) {
-  var isOutOfProcessDisabled = false;
+  var isOutOfProcessDisabled = true;
```
Comment (benfrancis) [inline apps/system/js/browser_config_helper.js]: What is this change for?

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=91c28e67e961ff300fffc2e0f2c2e519b4ed3e6e

**Pull request merged**